drools/drools-core/src/main/java/org/drools/core/time/SelfRemovalJobContext.java

```diff
@@ -47,9 +49,8 @@ public JobHandle getJobHandle() {
         return jobContext.getJobHandle();
     }
 
-    @Override
     public InternalWorkingMemory getWorkingMemory() {
-        return jobContext.getWorkingMemory();
+        throw new UnsupportedOperationException("delegate.getWorkingMemory()");
```

Comment:
is this still being used?

Reply:
right that's pretty ugly. I wanted to decouple JobContext from their WorkingMemory dependency and that's a dangling leftover. I'll look into it